src/config/cryptocurrencies.js

```diff
@@ -32,8 +32,10 @@ const supported: CryptoCurrencyIds[] = [
   'bitcoin_testnet',
 ]
 
-export const listCryptoCurrencies = memoize((withDevCrypto?: boolean) =>
-  listCC(withDevCrypto)
-    .filter(c => supported.includes(c.id))
-    .sort((a, b) => a.name.localeCompare(b.name)),
+export const listCryptoCurrencies = memoize(
```

Comment:
can you make sure memoize works with more than one param?

Reply:
uhmm damn, I'll check